app/routes/application.js

```diff
@@ -49,17 +49,29 @@ export default Route.extend(ApplicationRouteMixin, {
       }
     }
 
+    const pagesPromise = this.store.query('page', {
+      sort: 'index'
+    });
+
+    const settingsPromise = this.store.queryRecord('setting', {});
```

Comment:
Was this required? Settings are already injected in the application 🤔

Reply:
OK, then use it here